File: module-2/angular26-spring06/src/app/model/person.ts
export class Person {

  public id: number;
  public name: string;
  public age: number;

  constructor(obj?: any) {
    this.id   = obj && obj.id ? obj.id : null;
    this.name = obj && obj.name ? obj.name : null;
    this.age  = obj && obj.age ? obj.age : null;
  }
}

File: module-3/docker04/java-angular-postgres/spring-10/src/main/java/com/archsoft/model/Person.java
package com.archsoft.model;

import lombok.*;

import javax.persistence.*;
import javax.validation.constraints.NotBlank;
import javax.validation.constraints.NotNull;
import java.io.Serializable;

@Entity
@Table(name = "TB02_PERSON")
@Getter
@Setter
@ToString
@EqualsAndHashCode
@NoArgsConstructor
@AllArgsConstructor
public class Person implements Serializable {

    private static final long serialVersionUID = 4087182321318240446L;

    @Id
    @Column(name = "PERSON_ID")
    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "gen_person")
    private Long id;

    @NotBlank(message = "Nome requerido")
    @Column(name = "PERSON_NAME", nullable = false, length = 50)
    private String name;

    @NotNull(message = "Idade requerida")
    @Column(name = "PERSON_AGE", nullable = false)
    private Integer age;
}
